src/richie/plugins/section/choices.py

```diff
@@ -0,0 +1,18 @@
+"""
```

Comment:
In fact I would make the same comments as on PR https://github.com/openfun/richie/pull/497 so let's discuss and agree on something for the other one first.